src/app/policies/components/PolicyList.vue

```diff
@@ -0,0 +1,253 @@
+                      appearance="secondary"
+                      size="small"
+                    >
+                      <template #icon>
+                        <KIcon
+                          color="var(--black-400)"
+                          icon="more"
+                          size="16"
+                        />
+                      </template>
+                    </KButton>
+                  </template>
+
+                  <template #items>
+                    <KDropdownItem
+                      :item="{
+                        to: {
+                          name: 'policy-detail-view',
+                          params: {
+                            mesh: route.params.mesh,
+                            policyPath: policyType.path,
+                            policy: row.name,
+                          },
+                        },
+                        label: t('common.collection.actions.view'),
+                      }"
+                    />
+                  </template>
+                </KDropdownMenu>
+              </template>
+            </AppCollection>
+          </div>
+        </template>
+      </KCard>
+    </div>
+  </AppView>
+</template>
+
+<script lang="ts" setup>
+import {
+  KAlert,
+  KButton,
+  KCard,
+  KDropdownItem,
+  KDropdownMenu,
+  KIcon,
+} from '@kong/kongponents'
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+
+import { PolicyCollection } from '../sources'
+import AppCollection from '@/app/application/components/app-collection/AppCollection.vue'
+import AppView from '@/app/application/components/app-view/AppView.vue'
+import RouteTitle from '@/app/application/components/route-view/RouteTitle.vue'
+import DocumentationLink from '@/app/common/DocumentationLink.vue'
+import type { MeshInsight, PolicyType } from '@/types/index.d'
+import { useEnv, useI18n } from '@/utilities'
+
+type ChangeValue = {
+  page: number
+  size: number
+  s: string
+}
+
+const { t } = useI18n()
+const env = useEnv()
+const route = useRoute()
+
+const props = defineProps<{
+  pageNumber: number
+  pageSize: number
+  policyTypes: PolicyType[]
+  currentPolicyTypePath: string
+  policyCollection: PolicyCollection | undefined
+  policyError: Error | undefined
+  meshInsight: MeshInsight | undefined
+}>()
+
+const emit = defineEmits<{
+  (event: 'change', value: ChangeValue): void
+}>()
+
+const policyType = computed(() => props.policyTypes.find((policyType) => policyType.path === props.currentPolicyTypePath) ?? props.policyTypes[0])
+
+const sortedPolicyTypeWrappers = computed(() => {
+  const policyTypeWrappers = props.policyTypes.map((policyType) => ({
+    policyType,
+    numberOfPolicies: props.meshInsight?.policies?.[policyType.name]?.total ?? 0,
+  }))
+
+  policyTypeWrappers.sort((policyTypeWrapperA, policyTypeWrapperB) => {
+    return policyTypeWrapperA.policyType.name.localeCompare(policyTypeWrapperB.policyType.name)
+  })
```

Comment:
I know this used to be the existing sorting. However, Does it make sense? Seems like this would change the order of items in the sidebar and make the UX poor as it would prevent folks from developing muscle memory no?

Reply:
Since this sorts by name, shouldn’t this always be a stable and predictable order of items until we introduce new policy types?

Would you prefer to not sort them here and rely on whatever order `/policies` returns?

Reply:
This no longer applies sorting.